src/seedu/addressbook/storage/StorageFile.java

```diff
@@ -27,11 +27,21 @@
 
     /** Default file path used if the user doesn't provide the file name. */
     public static final String DEFAULT_STORAGE_FILEPATH = "addressbook.xml";
+    public static final String MISSING_STORAGE_MESSAGE = "File storage is missing!";
 
     /* Note: Note the use of nested classes below.
      * More info https://docs.oracle.com/javase/tutorial/java/javaOO/nested.html
      */
-
+    
+    /**
+     * Signals that the original storage file is missing.
+     */
+    public static class FileMissingException extends Exception {
```

Comment:
good job in trying to create an exception class on your own but creating an exception nested in the storage class is debatable. Nevertheless, nice try, those are just some subjective matters, once you know how to do, you can just follow the standards fix by any company or party or even yourself (if you have your own startup woohoo!)